signer/core/types.go

```diff
@@ -109,5 +109,8 @@ func (args *SendTxArgs) toTransaction() *types.Transaction {
 		*l1BlockNumber = *args.L1BlockNumber
 	}
 
-	return types.NewTransaction(uint64(args.Nonce), args.To.Address(), (*big.Int)(&args.Value), (uint64)(args.Gas), (*big.Int)(&args.GasPrice), input, l1MessageSender, l1BlockNumber, args.QueueOrigin, args.SignatureHashType)
+	tx := types.NewTransaction(uint64(args.Nonce), args.To.Address(), (*big.Int)(&args.Value), (uint64)(args.Gas), (*big.Int)(&args.GasPrice), input)
+	txMeta := types.NewTransactionMeta(l1BlockNumber, 0, l1MessageSender, args.SignatureHashType, args.QueueOrigin, nil, nil)
```

Comment:
Timestamp of `0` here? To be honest based on the outdated `SendTxArgs` struct, I feel like this is not used in our codepath but need to double check

Reply:
Timestamp would've been `0` anyway, no?

Reply:
Yes this is true